### PR TITLE
show [ip|ipv6] interfaces cmds

### PR DESCRIPTION
**Pull request by nikos-github**
We have a requirement for show [ip|ipv6] interfaces cmds which will display all IP addresses from all scopes of all interfaces along with admin/oper states, which is similar to existing vendor output.

Change depends on https://github.com/Azure/sonic-buildimage/pull/1353

Output of commands is below. Unfortunately the PR editor is not displaying the alignment properly.

```
root@lnos-x1-a-asw01:/home/admin# show ip int
Interface    IPv4 address/mask    Admin/Oper
-----------  -------------------  ------------
Ethernet0    10.0.0.0/31          up/down
Ethernet4    10.0.0.2/31          up/down
Ethernet8    10.0.0.4/31          up/down
Ethernet12   10.0.0.6/31          up/down
Ethernet16   10.0.0.8/31          up/down
Ethernet20   10.0.0.10/31         up/down
Ethernet24   10.0.0.12/31         up/down
Ethernet28   10.0.0.14/31         up/down
Ethernet32   10.0.0.16/31         up/down
Ethernet36   10.0.0.18/31         up/down
Ethernet40   10.0.0.20/31         up/down
Ethernet44   10.0.0.22/31         up/down
Ethernet48   10.0.0.24/31         up/down
Ethernet52   10.0.0.26/31         up/down
Ethernet56   10.0.0.28/31         up/down
Ethernet60   10.0.0.30/31         up/down
Ethernet64   10.0.0.32/31         up/down
Ethernet68   10.0.0.34/31         up/down
Ethernet72   10.0.0.36/31         up/down
Ethernet76   10.0.0.38/31         up/down
Ethernet80   10.0.0.40/31         up/down
Ethernet84   10.0.0.42/31         up/down
Ethernet88   10.0.0.44/31         up/down
Ethernet92   10.0.0.46/31         up/down
Ethernet96   10.0.0.48/31         up/down
Ethernet100  10.0.0.50/31         up/down
Ethernet104  10.0.0.52/31         up/down
Ethernet108  10.0.0.54/31         up/down
Ethernet112  10.0.0.56/31         up/down
Ethernet116  10.0.0.58/31         up/down
Ethernet120  10.0.0.60/31         up/down
Ethernet124  10.0.0.62/31         up/up
docker0      240.127.1.1/24       up/down
eth0         172.25.11.43/24      up/up
lo           127.0.0.1/8          up/up
             10.1.0.32/32
root@lnos-x1-a-asw01:/home/admin# show ipv6 int
Interface    IPv6 address/mask                       Admin/Oper
-----------  --------------------------------------  ------------
Bridge       fe80::705b:cfff:fee9:c4d%Bridge/64      up/up
Ethernet124  fe80::2e0:ecff:fe3c:a0a%Ethernet124/64  up/up
eth0         fe80::2e0:ecff:fe3c:a0a%eth0/64         up/up
lo           ::1/128                                 up/up`
```

### CODE REVIEW

File: show/main.py [hunk 1]
```diff
@@ -7,6 +7,8 @@
 import os
 import subprocess
 import sys
+import netifaces
```
Comment (jleveque) [inline show/main.py]: Please keep imports ordered alphabetically, e.g.,

```
import netifaces
import os
import subprocess
import sys
```

File: show/main.py [hunk 2]
```diff
@@ -7,6 +7,8 @@
...
+from netaddr import *
```
Comment (jleveque) [inline show/main.py]: It appears you're only using `IPAddress` from netaddr. I feel you should do one of the following instead;

`import netaddr`, then call `netaddr.IPAddress()` throughout the file

or,

`from netaddr import IPAddress`

I lean toward the former.

File: show/main.py [hunk 3]
```diff
@@ -349,12 +351,95 @@ def mac(vlan, port):
...
+@ip.command()
+def interfaces():
+    """Show interfaces IPv4 address"""
+    header = ['Interface', 'IPv4 address/mask', 'Admin/Oper']
+    data = []
+
+    interfaces = natsorted(netifaces.interfaces())
+
+    for iface in interfaces:
+
```
Comment (jleveque) [inline show/main.py]: Delete empty line after `for` for consistent style.

File: show/main.py [hunk 4]
```diff
@@ -349,12 +351,95 @@ def mac(vlan, port):
...
+        ipaddresses = netifaces.ifaddresses(iface)
+
+        if netifaces.AF_INET in ipaddresses:
+
```
Comment (jleveque) [inline show/main.py]: Delete empty line after `if` for consistent style.

File: show/main.py [hunk 5]
```diff
@@ -382,12 +467,51 @@ def protocol():
...
+@ipv6.command()
+def interfaces():
+    """Show interfaces IPv6 address"""
+    header = ['Interface', 'IPv6 address/mask', 'Admin/Oper']
+    data = []
+
+    interfaces = natsorted(netifaces.interfaces())
+
+    for iface in interfaces:
+
```
Comment (jleveque) [inline show/main.py]: Delete empty line after `for` for consistent style.

File: show/main.py [hunk 6]
```diff
@@ -382,12 +467,51 @@ def protocol():
...
+        ipaddresses = netifaces.ifaddresses(iface)
+
+        if netifaces.AF_INET6 in ipaddresses:
+
```
Comment (jleveque) [inline show/main.py]: Delete empty line after `if` for consistent style.

File: show/main.py (hunk 2 above)
Comment (nikos-github) [reply show/main.py]: Done

File: show/main.py (hunk 4 above)
Comment (nikos-github) [reply show/main.py]: Done.

File: show/main.py (hunk 3 above)
Comment (nikos-github) [reply show/main.py]: Done.

File: show/main.py (hunk 1 above)
Comment (nikos-github) [reply show/main.py]: Done.

**Comment by nikos-github** [comment]
@jleveque Comments addressed.

**Comment by jleveque** [comment]
Merge pending on https://github.com/Azure/sonic-buildimage/pull/1353

**Comment by lguohan** [comment]
retest this please

**Pull request merged**